### PR TITLE
Latest Dependencies

### PR DESCRIPTION
**Pull request by sandersky**
Use latest dependencies, most notably `karma`, `eslint`, and `postcss` instead of `autoprefixer`

### CODE REVIEW

File: src/github.js [hunk 1]
```diff
@@ -199,21 +197,21 @@ ns.bumpFiles = function (bump) {
     var v = versiony.from('package.json').indent(jsonIndent);
 
     switch (bump) {
-        case 'major':
-            v.newMajor();
-            break;
+    case 'major':
```
Comment (job13er) [inline src/github.js]: Did the new lint complain about indented `case` within `switch`? I've always preferred them indented myself...

**Comment by job13er** [comment]
Looks good :+1:

File: src/github.js (hunk 1 above)
Comment (sandersky) [inline src/github.js]: It did

Comment (job13er) [inline src/github.js]: Looks like we can configure that (if the team agrees with me, which I don't think they do):
http://eslint.org/docs/rules/indent#options

**Pull request merged**